src/collections/blog/2024/10-22-meet-the-maintainer-aadhitya-amarendiran/post.mdx

```diff
@@ -0,0 +1,197 @@
+  <span>Aadhitya:</span>
+  <p>
+    It was around the year 2020 when I was a sophomore and explored many things in the field of open source. I got to know about Meshery via the LFX program and tried applying for it, though I was a newcomer at that time. Later, I learned more about Layer5, started attending the community call, and met the community. The community members helped me a lot wherever I got stuck as a newcomer, which made me learn new things and involve myself in the project, which later helped me grow. Oh, of course, great learning sessions from Lee during development and community calls.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    You’re a Meshery Maintainer and have been for some long time now. What does
+    being a Meshery maintainer mean to you?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    Three things come into my mind: Learning lots of new things, Challenging yourself to your limits and being a helpful navigator for contributors. During my time as a newcomer, I started out with a simple readme fix PR in the Meshery project, which I thought would cause less impact. But the maintainers accepted my PR though it’s a very small one. That instilled a feeling in me that I should give back to the community by helping newcomers and contributors whenever they are stuck in work.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>Have you worked with any other open source project? How does Layer5 compare?</p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    Not a lot, but I worked on quite a few open source projects. Layer5 is one of the best places to start if you are new to open source. By being involved in the community, you will feel and understand the spirit of open source.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    <Link to="/projects">Layer5 projects</Link> have a number of active, open
+    source projects. You’ve been consistently contributing to a few of them.
+    Which one(s) are you currently focusing on?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+    I currently work on Meshery, as it piqued my interest during my initial days. I also work on the Meshery-SMP GitHub action project as well.
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>What’s the coolest Meshery demo you have done/seen?</p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>
+   I’ve seen a lot of demos but the coolest one for me is the Meshery Docker extension where you can start and use Meshery right from DockerHub!
+  </p>
+</div>
+
+<div class="interviewer">
+  <span>Vivek:</span>
+  <p>
+    What is your favorite Meshery CLI Command?
+  </p>
+</div>
+
+<div class="interviewee">
+  <span>Aadhitya:</span>
+  <p>Oof! That’s a tricky one. But my favorite one is definitely <b>mesheryctl perf</b></p>
```

Comment:
`mesheryctl perf` 
I wonder if it makes sense to wrap these commands in `<code></code>` instead :thinking: